Vectra/vectra_cognito_detect/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,126 @@
+[
+  {
+    "value": "{host.ip} set {vectra.detection.name}",
+    "conditions": [      
+      {
+         "field": "event.action",
+         "value": "INFO"
+      },
+      {
+         "field": "host.ip",
+      },
+      {
+         "field": "vectra.detection.name",
+      }      
+    ],
+    "relationships": [
+      {
+        "source": "host.ip",
+        "target": "destination.ip",
+        "type": "set"
+      }
+    ]
+  },
+  {
+    "value": "{vectra.history.host_access.name} logged on to {observer.ip} with {vectra.detection.last_type}",
+    "conditions": [
+      {
+         "field": "vectra.history.host_access.name",
+      },
+      {
+         "field": "observer.ip",
+      },
+      {
+         "field": "vectra.detection.last_type",
+      }  
+    ],
+    "relationships": [
+      {
+        "source": "vectra.history.host_access.name",
```

Comment:
To be used as an edge of a relationship, The declaration of the custom field should declare an observable property like [this one](https://github.com/SEKOIA-IO/intake-formats/blob/ce500dc94685f1c0b5fa7c5dc801fc4c7923585c/AWS/aws-cloudtrail/_meta/fields.yml#L90-L93)